.changeset/friendly-buses-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-tokens": major
```

Comment:
The tokens update will also include css variable updates for the form package (related to the RadioGroup and CheckboxGroup components)!